### DISCUSSION TITLE
第6节课作业（23班）

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#425] [作业]
作业提交

**Comment by PlateAss** [comment]
https://plateass.github.io/work6

**Comment by yaoyaotaba** [comment]
https://zhuanlan.zhihu.com/p/679044793

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/tFVER

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection6-OpenCompass-and-LLM-Evaluation/
